src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+type TodoCondition = 'neutral' | 'seving' | 'editing';
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [todoCondition] = useState<TodoCondition>('neutral');
```

Comment:
Consider removing unused state variable `todoCondition` or using it to control the rendering of the component. 🤔

Reply:
`# ❗️❗️❗️<br>Please implement ONLY todos loading, errors and filtering. <br><br>All the rest will be implemented in the next tasks<br>❗️❗️❗️`

I will add `setTodoCondition` on next step...